bdp_app/ios/Flutter/flutter_export_environment.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
```

Comment:
Wie du hier liest, sollte das nicht mit in die Version Control... Also am besten wieder rausnehmen mit:
`git rm --cached bdp_app/ios/Flutter/flutter_export_environment.sh`
und dann in die Datei .gitignore eine neue Zeile einfügen mit Inhalt:
`bdp_app/ios/Flutter/flutter_export_environment.sh`